ksql-engine/src/main/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactory.java

```diff
@@ -98,6 +100,10 @@ static void configureSslFactory(final KsqlConfig config, final SslFactory sslFac
   }
 
   public SchemaRegistryClient get() {
+    if (schemaRegistryUrl.equals("")) {
```

Comment:
Trim the value before comparing it. If the value is "   ", then this check will think there is a URL specified. I recommend trimming the value once in the constructor when assigning the string to the `schemaRegistryUrl` so we don't have to trim it every time we use it.

Reply:
done